phpcs.xml.dist

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="LiteSpeed_Cache" xsi:noNamespaceSchemaLocation="lsc.xsd">
+	<description>Apply LiteSpeed Cache Coding Standards to all plugin files</description>
+
+	<!--
+	#############################################################################
+	COMMAND LINE ARGUMENTS
+	https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset
+	#############################################################################
+	-->
+
+	<!-- Only scan PHP files -->
+	<arg name="extensions" value="php"/>
+
+	<!-- Cache scan results to use for unchanged files on future scans -->
+	<arg name="cache" value=".cache/phpcs.json"/>
+
+	<!-- Set memory limit to 512M
+		 Ref: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#specifying-phpini-settings
+	-->
+	<ini name="memory_limit" value="512M"/>
+
+	<!-- Remove unwanted prefix from filepaths -->
+	<arg name="basepath" value="./"/>
+
+	<!-- Check max 20 files in parallel -->
+	<arg name="parallel" value="20"/>
+
+	<!-- Show sniff codes in all reports -->
+	<arg value="ps"/>
+	
+	<arg name="parallel" value="8"/>
```

Comment:
Remove this and update L26-27 instead.

Reply:
My mistake. I did not saw it's duplicated. Left only one line